packages/server-wallet/Procfile

```diff
@@ -0,0 +1 @@
+hub: node ./lib/hub/server.js
```

Comment:
Let's get rid of this file. We are using docker containers now (as opposed to Heroku slugs).